Digipost/src/main/java/no/digipost/android/authentication/Secret.java

```diff
@@ -0,0 +1,12 @@
+package no.digipost.android.authentication;
```

Comment:
This will most likely create a merge-conflict when/if merged with repo. Its less time consuming to have it this way, instead of evry user creates the same file with almost the same info.

Reply:
Enig :)